internal/service/appconfig/configuration_profile.go

```diff
@@ -63,6 +63,12 @@ func ResourceConfigurationProfile() *schema.Resource {
 				Optional:     true,
 				ValidateFunc: verify.ValidARN,
 			},
+			"type": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Default:      "AWS.Freeform",
```

Comment:
This default isn't explicitly documented by AWS, but is the observed behaviour

Reply:
That's fine; there are plenty of other resource attributes where we have "discovered" defaults. Thanks.